runtimes/runtimes/operational-telemetry/aws-spans-exporter.ts

```diff
@@ -29,6 +29,17 @@ export class AwsSpanExporter implements SpanExporter {
             resultCallback({ code: ExportResultCode.FAILED })
             return
         }
+        if (this.telemetryService.getTelemetryStatus() === TelemetryStatus.Pending) {
```

Comment:
Really a corner case, but should we make sure we do not have memory leak because of this, if logic with status change is not working?

Reply:
Would a queue size limit be an effective solution here?